### PR TITLE
added license

### PR DESCRIPTION
**Pull request by justinalexandershaw**
@sorenrood What do you think of this license?

### CODE REVIEW

**Comment by justinalexandershaw** [comment]
See: 
- https://softwareengineering.stackexchange.com/questions/253925/how-to-avoid-being-forked-into-oblivion-by-a-more-powerful-contributor
- https://softwareengineering.stackexchange.com/questions/68134/best-existing-license-for-closed-source-code

**Comment by justinalexandershaw** [comment]
[This](https://opensource.stackexchange.com/a/210) is one of the best articles I could find.

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Copyright © 2020 Justin Shaw
+All Rights Reserved.
```
Comment (justinalexandershaw) [inline LICENSE.txt:2]: Do you think this is enough?

**Review by justinalexandershaw — COMMENTED** [review]
What do you think?

File: LICENSE.txt (hunk 1 above)
Comment (sorenrood) [reply LICENSE.txt:2]: I may add something about the project it's being used for. I read the links you included. Do you plan on this project being open-source? Or private?

**Pull request merged**